src/In.ProjectEKA.OtpService/Notification/NotificationService.cs

```diff
@@ -34,5 +43,14 @@ private static string GenerateConsentRequestMessage(JObject notificationContent)
                 $"To view request, please tap on the link: {content.DeepLinkUrl}";
             return message;
         }
+
+        private static string GenerateCmidRecoveryMessage(JObject notificationContent)
+        {
+            var cMidContent = notificationContent.ToObject<CMidContent>();
```

Comment:
Why M caps here? Then should not your CmidRecovered =› be CMidRecovered, why id is not treated as disjointed to the word CM

Reply:
Refactor the code where ever is required